index.html

```diff
@@ -8,9 +8,39 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
     />
     <link rel="stylesheet" type="text/css" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
   </head>
   <body>
-    <div class="js-grid-container grid-container"></div>
+    <header class="header">Etch a sketch</header>
```

Comment:
- Use title case for the heading.
- Use `h1` as a semantically better element. Read about `header` vs `h1`, `h2`, etc tags.